neqo-transport/src/cc.rs

```diff
@@ -29,6 +29,8 @@ pub struct CongestionControl {
     bytes_in_flight: usize,
     congestion_recovery_start_time: Option<Instant>,
     ssthresh: usize,
+
+    oldest_lost_pkt_time: Option<Instant>,
```

Comment:
Let's put some comments in here, because this isn't the oldest lost packet, because that would never be updated once it was first set.

This is the time of the first lost packet that follows the largest acknowledged.